charts/stable/collabora-online/questions.yaml

```diff
@@ -272,7 +272,7 @@ questions:
                 label: "Privileged mode"
                 schema:
                   type: boolean
-                  default: false
+                  default: true
```

Comment:
This is technically even more unsafe than running as root.

Reply:
I know, but without it, there is this errors
`sh: 1: /usr/bin/coolmount: Operation not permitted`

Reply:
Running as root results in the app crashing
Demanding to NOT run as root

Reply:
> I know, but without it, there is this errors `sh: 1: /usr/bin/coolmount: Operation not permitted`

I think you mean "loolmount", which is not actually required for the container to run.
(it will try another method instead.)

Reply:
https://github.com/CollaboraOnline/online/issues/779

Reply:
its actuall `coolmount` now :D 
But yea, if it chooses another way, then i'll leave privileged out!